web/pandas_web.py

```diff
@@ -258,6 +259,21 @@ def extend_base_template(content: str, base_template: str) -> str:
     return result
 
 
+def build_interactive_terminal(source_path: str, target_path: str) -> int:
```

Comment:
I think this makes sense conceptually, but I'd probably build this directly in the CI. I see two advantages:
- It's simpler
- When working on changes in the website, we don't need to rebuild this every time, which I guess it takes some time.

Reply:
Sounds good. Normally in this particular case the build should be relatively fast since it mostly unpacks the assets from the jupyterlite archive.